thrift/src/main/thrift/rpc.thrift

```diff
@@ -269,6 +269,15 @@ struct TSInsertRecordsOfOneDeviceReq {
     6: optional bool isAligned
 }
 
+struct TSInsertStringRecordsOfOneDeviceReq {
```

Comment:
Since you change rpc.thrift,  you should also update rpc change list doc.

Reply:
I've updated this rpc change list doc.